scripts/exhafs_post.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/ksh
+#!/bin/sh
```

Comment:
@BantwaleEnyew-NOAA Please double check if the following code section in this script works fine with /bin/sh. Previously, somehow it did not work. Thanks!
```

# In /bin/sh this cat part does not work
# Need to figure it out why. Use ksh for now.
cat>itag<<EOF
${INPdir}/dynf${FHR3}.nc
netcdf
grib2
${YYYY}-${MM}-${DD}_${HH}:00:00
FV3R
${INPdir}/phyf${FHR3}.nc
&NAMPGB
KPO=47,PO=1000.,975.,950.,925.,900.,875.,850.,825.,800.,775.,750.,725.,700.,675.,650.,625.,600.,575.,550.,525.,500.,475.,450.,425.,400.,375.,350.,325.,300.,275.,250.,225.,200.,175.,150.,125.,100.,70.,50.,30.,20.,10.,7.,5.,3.,2.,1.,
/
EOF
```

Reply:
Merging the PR and will test after.